pkg/grpc/grpc.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
why did you need to move `grpc` from storage? `pkg` is our general-purpose modules, this does not really fit here because it's coupled with business-specific storage APIs

Reply:
I anticipated that you would request these sub-packages of `plugin/storage/grpc` be moved to `pkg/grpc` because both `plugin/storage/grpc` and `plugin/metrics/grpc` depend on them. I'll move them back.

Reply:
Rather than making the PR more complicated with an additional file-move PR, I've removed the commit that moved `plugin/storage/grpc/config` and `plugin/storage/grpc/shared` to `pkg/grpc`.